src/Agent.Listener/Capabilities/AgentCapabilitiesProvider.cs

```diff
@@ -34,6 +35,10 @@ public Task<List<Capability>> GetCapabilitiesAsync(AgentSettings settings, Cance
 #if OS_WINDOWS
             Add(capabilities, "Agent.OSVersion", GetOSVersionString());
             Add(capabilities, "Cmd", Environment.GetEnvironmentVariable("comspec"));
+
+            //This capability gets set irrespective of agent being run in auto-logon mode or not.
+            var configurationManager = HostContext.GetService<IConfigurationManager>();
```

Comment:
try load setting directly from ConfigurationStore instead of ConfigurationManager. it's weird ConfigurationManager just a wrapper over configstore in some case, like this.